tests/presets/celery_test.py

```diff
@@ -21,15 +22,32 @@ def test_worker_config():
             'redis://')
 
 
-def test_workeron_loaded():
+def test_worker_on_loaded():
     conf = WorkerConfiguration({
         'worker': {
-            'on_loaded': "assert app(self) == 'ok'",
+            'on_loaded': f"assert app.main == '{__name__}'\n"
```

Comment:
How about using a docstring If newline is required?

```
f"""assert app.main == '{__name__}'
app.main = 'ok'"""
```

Reply:
With the docstring it will look like ...

```python
...
        'worker': {
            'on_loaded': f"""assert app.main == '{__name__}'
app.main = 'ok'""",
        }
...
```

... because of the indentation.